sklearn/linear_model/_coordinate_descent.py

```diff
@@ -224,14 +224,29 @@ def lasso_path(
 
         (1 / (2 * n_samples)) * ||y - Xw||^2_2 + alpha * ||w||_1
 
+    .. math::
+        \\frac{1}{2n_{\\operatorname{samples}}}
+        \\vert \\vert y - Xw \\vert \\vert^2_2 +
+        \\alpha \\vert \\vert w \\vert \\vert_1
+
     For multi-output tasks it is::
 
         (1 / (2 * n_samples)) * ||Y - XW||^2_Fro + alpha * ||W||_21
 
-    Where::
+    .. math::
+        \\frac{1}{2n_{\\operatorname{samples}}}
+        \\vert \\vert Y- XW \\vert \\vert^2_F +
+        \\alpha \\vert \\vert W \\vert \\vert_{2,1}
+
+    where :math:`\\vert\\vert W \\vert\\vert_F` is the Frobenius norm of :math:`W`,
+    and::
```

Comment:
I don't love that these are basically rendered twice. Ideally we'd have only one version of them, not both, but I do see that it's less readable in latex format unrendered.

Maybe a good solution would be to rst-commen (with `.. `) the "code form" and only have the latex form rendered.

Reply:
I recall that this is a long standing debate. I was always more on the side of no-latex because I don't find it readable when looking at my IDE docstring. I don't know if modern IDE, is actually translating latex to HTML view nowadays?

Reply:
Yeah I agree that I don't think it needs to be here twice. Less opinionated on whether its the code or latex one that we should have.

As @glemaitre said, I guess one view is for no latex in docstring, and have this in stuff in the user guide instead?